<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanLoginWidget</class>
 <widget class="QWidget" name="ScanLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanLoginWidget</string>
  </property>
  <widget class="QLabelHeader" name="labelHeader">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>160</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNikeName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阿玛尼好</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelChangeUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>切换账户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>250</y>
     <width>261</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLabelHeader</class>
   <extends>QLabel</extends>
   <header>..\SharedLib\BaseUI\include\qlabelheader.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
